src/web_app_skeleton/.github/workflows/ci.yml.ecr

```diff
@@ -0,0 +1,114 @@
+name: <%= project_name %> CI
+
+on:
+  push:
+    branches: "*"
+  pull_request:
+    branches: "*"
```

Comment:
We should actually be able to consolidate this according to [the docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#on)

```suggestion
on: [push, pull_request]
```

Reply:
My version  is more flexible and self descriptive, and allow easy change "*" to master if need, e.g. like here
https://github.com/luckyframework/lucky_cli/blob/8be29ea4af81fb2ae027093ea8c38d56cb0cab11/.github/workflows/ci.yml#L4-L7

Reply:
That's a good point. I think keeping it as-is makes it easier to customize so can be left as-is

Reply:
Sounds good to me!